core-services/secret-mirroring/_mapping.yaml

```diff
@@ -342,3 +342,9 @@ secrets:
   to:
     namespace: ci
     name: rbac-permissions-operator-codecov-token
+- from:
+    namespace: ocpappsvc-secrets
+    name: redhat-developer-services-binding-operator-test-quay-credentials
+  to:
+    namespace: ci
```

Comment:
The secrets to be used by test steps should be synced into `test-credentials` namespace (https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-custom-credentials)

Reply:
Thx